app/src/docs/components/DocsLayout/docsLayout.pcss

```diff
@@ -33,10 +30,6 @@
       border-radius: 4px;
       background-color: #EAEAEA;
     }
-
-    code {
-      font-size: 14px;
-    }
```

Comment:
Styling in this file is either redundant now or too specific (note dropped `!important`). FTW!